vendor/knative.dev/eventing/pkg/apis/duck/v1alpha1/channelable_types.go

```diff
@@ -95,6 +95,19 @@ func (c *Channelable) Populate() {
 			},
 		},
 		SubscribableTypeStatus: SubscribableTypeStatus{
+			DeprecatedSubscribableStatus: &SubscribableStatus{
```

Comment:
hrm... is that right ? @Harwayne @mattmoor  ?

Reply:
Looks correct to me.